src/validator_test.py

```diff
@@ -0,0 +1,154 @@
+import datetime
+import logging
+import os
+import random
+
+import pickle
+
+import torch
+from substrateinterface import Keypair  # type: ignore
+
+import src.utils as utils
+from src.workload import Workload
+from src.validator import ComputeValidator
+
+# from reward import Reward
+time_str = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
```

Comment:
Better to include formatting strings in another file. These are likely to be re-used in multiple places and really easy way to introduce bugs if typed out manually